internal/event/event_type.go

```diff
@@ -16,12 +16,13 @@ const (
 	AuditType       Type = "audit"       // AuditType represents audit events
 	ErrorType       Type = "error"       // ErrorType represents error events
 	SystemType      Type = "system"      // SysType represents system events
+	StorageType     Type = "storage"     // StorageType represents session recording events
```

Comment:
Should this maybe say session recording storage events? Or just storage events?

Reply:
I was wondering this too. The reason why I chose to keep it vague is because storage could potentially be used for more than just session recordings. I'm not sure if that's a good enough reason to keep it vague though. I'm happy to change it to `SessionRecordingStorageType Type = "session_recording_storage_type"`

Reply:
Ah sorry, I meant the comment indicating that it's a session recording event; should the comment just say that it represents a storage event?

Reply:
Good catch, sorry for misreading your original feedback. I changed it to say `StorageType represents storage events`